my-app/src/AddBox/AddBox.jsx

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import Autosuggest from 'react-autosuggest';
+
+//import isMobile from 'ismobilejs';
+
+//const focusInputOnSuggestionClick = !isMobile.any;
+const uniqLabelFast = a => {
+  /**
+   * Quickly makes a list tags without repeating labels
+   * @param a {[Tag]} list of tags
+   */
+
+    var seen = {};
+    var out = [];
+    var len = a.length;
+    var j = 0;
+    for(var i = 0; i < len; i++) {
+         var item = a[i].label;
+         if(seen[item] !== 1) {
+               seen[item] = 1;
+               out[j++] = a[i];
+         }
+    }
+    return out;
+}
+
+const escapeRegexCharacters = str => str.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+
+const getSuggestionValue = (object) => {
```

Comment:
Is there a more descriptive name that can be given to the variable 'object'?  It looks to me like it should be something with either a .label or a .name property.  Does that mean its either a person or a tag?  A comment might be okay here to explain the purpose, or if the variable name is really verbose then maybe no comment is needed.

I think it is functionally equivalent to say `const getSuggestionValue = o => o.label || o.name || null`